### PR TITLE
update docs and comments

### PR DESCRIPTION
**Pull request by MaxUlysse**
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Added
 
+- [#230](https://github.com/nf-core/sarek/pull/230) - Add `ignore_soft_clipped_bases` option for `Mutect2` [#218](https://github.com/nf-core/sarek/issues/218)
+
 ### Changed
 
 - [#237](https://github.com/nf-core/sarek/pull/237) - Switch `bwa 0.7.17` for `bwa-mem2 2.0`
```
Comment (MaxUlysse) [inline CHANGELOG.md:15]: - [#255](https://github.com/nf-core/sarek/pull/255) - Update docs

File: nextflow.config [hunk 2]
```diff
@@ -66,7 +66,7 @@ params {
   no_strelka_bp = null // Strelka will use Manta candidateSmallIndels if available
   pon = false // No default PON (Panel of Normals) file for GATK Mutect2 / Sentieon TNscope
   pon_index = false // No default PON index for GATK Mutect2 / Sentieon TNscope
-  ignore_soft_clipped_bases = null // GATK Mutect2
+  ignore_soft_clipped_bases = null // enable --dont-use-soft-clipped-bases for GATK Mutect2
```
Comment (apeltzer) [inline nextflow.config:69]: Is the function already used in the mutect2 step or just defined here?

Comment (MaxUlysse) [reply nextflow.config:69]: in used since we merged #230 actually, just forgot to add some docs

**Comment by MaxUlysse** [comment]
> Guess fine, but see my comment.

Replied ;-)

>Resolved the merge conflict for u.

Thanks,
I've been looking for updating PRs since they changed the GUI, where is it now?

**Comment by apeltzer** [comment]
> I've been looking for updating PRs since they changed the GUI, where is it now?

Its down below the checks now, at least was for me :-)

**Pull request merged**